libraries/RC_Channel/RC_Channel.cpp

```diff
@@ -1859,27 +1859,24 @@ RC_Channel *RC_Channels::find_channel_for_option(const RC_Channel::AUX_FUNC opti
 // duplicate_options_exist - returns true if any options are duplicated
 bool RC_Channels::duplicate_options_exist()
 {
-    uint8_t auxsw_option_counts[512] = {};
+    Bitmask<512> used_auxsw_options;
```

Comment:
```suggestion
    Bitmask<(uint16_t)RC_Channel::AUX_FUNC::AUX_FUNCTION_MAX> used_auxsw_options;
```

Reply:
Huh.  We have an _MAX entry now.  Interesting, I'd forgotten that....

Done.  I'm not *really* sure it's a win.  Yeah, the code won't get out of date with the number of options, but that won't happen for a while.  And obviously the stack space isn't an issue :-)

With this formulation - pity the poor bugger in his fork who decides to use option number 40,000 for his extra-special-super-secret AUXFUNC... at least he's likely to get a frame-size compilation failure rather than a runtime crash, but still an interesting issue.

Reply:
That is fair, I had considered suggesting a static_assert that the bit mask didn’t end up over 512 bytes as we had decent evidence that much stack space was safe, but forgot about it. We’ll see what the dev call thinks.